packages/edit-site/src/components/page-templates/index.js

```diff
@@ -78,9 +78,7 @@ const DEFAULT_VIEW = {
 		field: 'title',
 		direction: 'asc',
 	},
-	// All fields are visible by default, so it's
-	// better to keep track of the hidden ones.
-	hiddenFields: [ 'preview' ],
```

Comment:
Question: what if you hide the preview field, but it's required in another layout?

Reply:
It's still shown in the other layout (there's special treatment for media and primary fields). That said, we've been discussing with @jameskoster and @oandregal and we'll be following-up with a PR where the visible fields and fields rendering config resets when you switch layout. We found that it makes little sense to keep these when you switch layouts (as opposed to filters...)

Reply:
Related https://github.com/WordPress/gutenberg/issues/57596#issuecomment-2134807657